stan-models/AR1.R

```diff
@@ -0,0 +1,158 @@
+library(dplyr)
+library(tidyr)
+library(rstan)
+devtools::load_all()
+
+start_yr <- 1982
+end_yr <- 2013
+min_num_yrs <- 25
+last_training_year <- 2003
+richness_w_env <- get_richness_ts_env_data(start_yr, end_yr, min_num_yrs) %>%
```

Comment:
Hmm.  This is the same code that's being used in `forecasting-bbs-R.ipynb`, I think.  Want to make a separate Issue to discuss it?